examples/passive_scalar/benchmark/gaussian2d/anisotropic/BGK_EmodelCorr/seeder.lua

```diff
@@ -0,0 +1,93 @@
+require 'args'
+
+folder    = 'mesh/'
+comment   = "gaussian_pulse"
+
+minlevel  = level_usr
+print("level of treelm: ", minlevel)
```

Comment:
I don't think we should have a print here. You anyway already split up your configuration files into multiple. If you want to have one for running lua, providing some infos, I'd suggest to have an additional lua file with something like:

```
require `seeder`
print("level of treelm: ", minlevel)
```